### ISSUE TITLE
Visual Editor

### ISSUE THREAD

**Issue by ablatiii** [GM-Alex/user-access-manager#29]
A user who has had author and editor rights to a page suddenly about a week and a half ago no longer is able to edit the page with the visual editor.  Nothing has changed that I can tell.  User is set to a group that has access to the page, he can edit the page, however, once the editor loads, it does not load the Visual editor and all text in the text editor is white, thus unreadable unless highlighted.  Any ideas?

**Comment by GM-Alex** [comment]
Are you sure that the user access manager is the reason? Could you try it with the incognito mode of your browser? Seems for me like a JS error.

**Comment by ablatiii** [comment]
Just tried it while in incognito mode and it still occurs.  I'm not positive that it is UAM, however, disabling UAM seems to fix the issue.  Only when it is enabled does the issue occur.

**Comment by GM-Alex** [comment]
Hm... sounds strange. If you have a test environment could you try to disable all plugins expect UAM, if it's still there it seems a problem with UAM itself, if not start to enabling the other plugins and check after every plugin.

**Comment by ablatiii** [comment]
Okay, my apologies.  I tried first deactiving all EXCEPT UAM and the problem still occurred.  However, I then deactivated UAM as well and the issue still was occurring.  So I'm unsure where to go from here.  What had been working is no longer working, even though nothing in the environment had changed.  From what I just tested, only Administrator's are able to see the visual editor and Editor's are getting the white text on white background thus looking like nothing is there in the visual editor.  But from that recent test I'd say it doesn't look like it's a UAM issue.  Thanks for the help.
